### PR TITLE
2023H1 Roadmap

### PR DESCRIPTION
**Pull request by hone**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,71 +1,39 @@
...
-### Test API
-- **Outcomes**: 
-  - A specification defining an API for setting up and running tests in a buildpacks-based environment.
-- **Personas**:
-  - Application Developers
-  - Buildpack Authors
-- **Status**: Not started
+## Release Base Image Extension
+* Owner: @natalieparellano 
+* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0105-dockerfiles.md)
```
Comment (natalieparellano) [inline ROADMAP.md:7]:
```suggestion
* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0105-dockerfiles.md), [tracking issue](https://github.com/buildpacks/rfcs/issues/224)
```

File: ROADMAP.md [hunk 2]
```diff
@@ -1,71 +1,39 @@
...
 
-### Spec Refactor
-- **Outcomes**: 
-  - A restructured specification for Cloud Native Buildpacks that is more approachable, and easier to ammend.
-- **Personas**:
-  - Platform Authors
-  - Buildpack Authors
-  - Contributors
-- **Status**: Not started
+This started out as [Stack Buildpacks](https://github.com/buildpacks/rfcs/blob/main/text/0069-stack-buildpacks.md) and now Dockerfile Extensions. Significant work has already been done on this feature over the last year. This roadmap item is about seeing this work through with releasing phase 3 in both `lifecycle` and `pack`.
```
Comment (natalieparellano) [inline ROADMAP.md:9]:
```suggestion
This started out as [Stack Buildpacks](https://github.com/buildpacks/rfcs/blob/main/text/0069-stack-buildpacks.md) and is now Dockerfile Extensions. Significant work has already been done on this feature over the last year. This roadmap item is about seeing this work through with releasing phase 3 in both `lifecycle` and `pack`.
```

File: ROADMAP.md [hunk 3]
```diff
@@ -1,71 +1,39 @@
...
-  - Support for Inline Buildpacks.
-- **Personas**:
-  - Application Developers
-  - Buildpack Authors
-- **Status**: Not started
-- **RFCs**:
-  - [RFC-0048](https://github.com/buildpacks/rfcs/blob/main/text/0048-inline-buildpack.md)
+## Remove Stacks & Mixins
+* Owner: @jkutner
+* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)
```
Comment (natalieparellano) [inline ROADMAP.md:13]:
```suggestion
* Links: [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md), [tracking issue](https://github.com/buildpacks/rfcs/issues/219)
```

File: ROADMAP.md [hunk 4]
```diff
@@ -1,71 +1,39 @@
...
+* Owner: @hone
+* Links: [RFC](https://github.com/buildpacks/rfcs/pull/274)
+
+There has long been a desire for a "Test" support, but it's never been prioritized even though it's made the roadmap before. Not to be over ambitious, the first step is to get a RFC written and accepted.
+
+## Project Health
+* Owner: @samj1912
+* Links: [Buildpacks Community Organization RFC](https://github.com/buildpacks/rfcs/pull/273)
+
+Like other [CNCF projects](https://github.com/cncf/toc/issues?q=is%3Aissue+sort%3Aupdated-desc+%22health+of%22+-label%3A%22project+onboarding%22+-label%3A%22sandbox%22+), the project has been impacted by the VMware + Broadcom acquisition. The goal of this item is to improve the general health of the project and grow contributors back to our 2020 numbers. This inludes every team having a set of active set of maintainers and contributors, thus removing the TOC needing to step in for platform.
```
Comment (natalieparellano) [inline ROADMAP.md:27]:
```suggestion
Like other [CNCF projects](https://github.com/cncf/toc/issues?q=is%3Aissue+sort%3Aupdated-desc+%22health+of%22+-label%3A%22project+onboarding%22+-label%3A%22sandbox%22+), the project has been impacted by the VMware + Broadcom acquisition. The goal of this item is to improve the general health of the project and grow contributors back to our 2020 numbers. This includes every team having a set of active set of maintainers and contributors, thus removing the TOC needing to step in for platform.
```

**Review by natalieparellano — APPROVED** [review]

**Review by samj1912 — APPROVED** [review]

**Comment by AidanDelaney** [comment]
Can we pull in @natalieparellano's traking links before we land?

**Comment by hone** [comment]
@AidanDelaney done.

**Pull request merged**